guides/cookbook/incremental-adoption-guide.md

```diff
@@ -0,0 +1,146 @@
+
+If you have different backend format - EmberData provides you with builders for `REST`([`@ember-data/rest`](https://github.com/emberjs/data/tree/main/packages/rest)) and `ActiveRecord`([`@ember-data/active-record`](https://github.com/emberjs/data/tree/main/packages/active-record)).
+
+`@ember-data/request-utils` package contains a lot of useful utilities for building requests. You can read more about it in its [Readme](https://github.com/emberjs/data/tree/main/packages/request-utils#readme). It has request builders for all type of requests.
+
+## Step 5: Off you go! Start using new APIs
+
+Now you can start refactoring old code to use new APIs. You can start with the `findAll` method. It is the easiest one to refactor. Here is how you do it:
+
+```diff app/components/projects/list.ts
++ import { query } from '@ember-data/json-api/request';
+
+  loadProjects: Task<void, []> = task(async () => {
+-    const projects = await this.store.findAll('project');
+-    this.projects = [...projects];
++    const { content } = await this.store.request(query('project', {}, { host: config.api.host }));
++    this.projects = content.data;
+  });
+```
+
+You most likely would need to add Auth Handler to your request manager to add `accessToken` to your requests.
+Let's say you have your `accessToken` in the `session` service. Here is how you can add it to the request manager:
+
+```js
+import { inject as service } from '@ember/service';
+
+export default class AuthHandler {
+  @service session;
+
+  request({ request }, next) {
+    const headers = new Headers(request.headers);
+    headers.append(
+      'Authorization',
+      `Bearer ${this.session.accessToken}`,
+    );
+
+    return next(Object.assign({}, request, { headers }));
+  }
+}
+```
+
+You can read more about auth topic [here](../requests/auth.md).
+
+Another good thing to do is to configure default host and namespace for your requests. There is an utility for that out of the box of `@ember-data/request-utils` called [`setBuildURLConfig`](https://github.com/emberjs/data/blob/main/packages/request-utils/src/index.ts#L67). You can do it anywhere in your app theoretically, but we recommend doing it in the `app/app.js` file. Here is how you can do it:
+
+```diff app/app.js
+import Application from '@ember/application';
+import Resolver from 'ember-resolver';
+import loadInitializers from 'ember-load-initializers';
+import config from 'base-ember-typescript-app/config/environment';
++import { setBuildURLConfig } from '@ember-data/request-utils';
+
+export default class App extends Application {
+  modulePrefix = config.modulePrefix;
+  podModulePrefix = config.podModulePrefix;
+  Resolver = Resolver;
+}
+
+loadInitializers(App, config.modulePrefix);
```

Comment:
I'm not sure if this could happen (so maybe just ignore this), but if an initializer somehow uses ED, then `setBuildURLConfig` should be called before `loadInitializers`.